common/on_actions/SOV_black_revolution_cw.txt

```diff
@@ -354,4 +354,229 @@ on_actions = {
 			}
 		}
 	}
+
+	#Anarchist League Defensive
+	on_war_relation_added = {
+		effect = {
+			if = {
+				limit = {
+					FROM = {
+						OR = {
+							tag = SOV
+							is_in_faction_with = SOV
+						}
+					}
+					ROOT = {
+						NOT = {
+							is_in_faction_with = SOV
+							is_russian_state_russia2 = yes
+						}
+					}
+					SOV = {
+						has_completed_focus = SOV_we_are_stronger_together
+						has_government = anarchy
+						is_subject = no
+						is_faction_leader = yes
+					}
+				}
+				every_country = {
+					limit = {
+						OR = {
+							tag = SOV
+							is_in_faction_with = SOV
+						}
+						NOT = {
+							has_war_with = ROOT
+						}
+					}
+					ROOT = {
+						declare_war_on = {
+							target = PREV
+							type = defense_anarchist_league
+						}
+					}
+				}
+			}
+		}
+	}
+
+	#Anarchist League Offensive
+	#on_war_relation_added = {
```

Comment:
Remove ?

Reply:
We should keep it for the moment, I might look at an alternative after the test game